src/plugins/global-settings/color-schemes/index.php

```
@@ -191,6 +194,35 @@ public static function get_color_scheme_properties( $values = null ) {
 			return $_properties;
 		}
 
+		public function extend_color_scheme( $old_version, $new_version ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$new_version'. (undefined)

(UnusedFormalParameter)